src/context/computer_board_context.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, createContext, useEffect } from 'react';
```

Comment:
Nit: prefer `PascalCased` filenames to keep naming consistent

Reply:
Fixed!